backend/spec/jobs/notification_slack_job_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe NotificationSlackJob, type: :job do
```

Comment:
In general, I liked the tests, but I think that it's also important to check if, the `SlackService`, has been called, with correct params, what do you think about it?

Reply:
Yes, added a test to check if slack service call method has been called.